common/client.go

```diff
@@ -112,11 +112,21 @@ func (c *DatabricksClient) Delete(ctx context.Context, path string, request any)
 	return c.Do(ctx, http.MethodDelete, path, nil, request, nil, c.addApiPrefix)
 }
 
+// Delete on path
+func (c *DatabricksClient) DeleteWithResponse(ctx context.Context, path string, request any, response any) error {
```

Comment:
This might lead to confusion down the line as to which Delete method we should for the client. It's not immediately obvious if we should use Delete or DeleteWithResponse. It would be good to add a little more context in comments for this and PatchWithResponse

Reply:
Added comment